charters/MediaStrategyCharter.md

```diff
@@ -0,0 +1,42 @@
+# Jupyter Media Strategy Working Group
+
+## Purpose
+
+The mission of the Jupyter Media Strategy Working Group (JMS) is to ensure that communications in Jupyter official channels are strategic and benefit Project Jupyter. The Jupyter[ governance model ](https://jupyter.org/governance/standing_committees_and_working_groups.html)includes a distributed model of responsibilities. This means that by design Jupyter publications, social media posts, promotions and other media activity are meant to be community driven. The JMS is vendor neutral and will provide mechanisms to enable the community to speak publicly through the official Jupyter communication channels. The JMS will also help to improve Jupyter media activity by creating strategy and guidelines, serving as editors for existing media channels, and overseeing creation/delegation of new media channels.
```

Comment:
>  communications in Jupyter official

Nitpicking and It's obvious I think from the context, this is outward-facing channels, i.e we don't consider thigns like slack, or internal communicatino channels. I don't think this is needed, but you may want to clarify ?

Reply:
Thanks, we added "public" in front of media channels.